lib/eventboss/version.rb

```diff
@@ -1,3 +1,3 @@
 module Eventboss
-  VERSION = "1.3.4"
+  VERSION = "1.4.0"
```

Comment:
```suggestion
  VERSION = "1.3.4"
```

Reply:
actually why not bump a minor, @managr?

Reply:
Because it's not changing the api? Dunno, I'm not against it, but it's a tiny change :)